Mainnet/DividendToken/DividendToken/DividendToken.cs

```diff
@@ -116,7 +115,8 @@ public ulong GetDividends(Address address)
     public ulong GetTotalDividends(Address address)
     {
         var account = GetAccount(address);
-        return checked(GetWithdrawableDividends(address, account) + account.WithdrawnDividends) / TotalSupply;
+        var withdrawable = GetWithdrawableDividends(address, account) / TotalSupply;
+        return withdrawable + account.WithdrawnDividends;
```

Comment:
Are the latest updates correcting the previous code here? 

Initially had `(a + b) / c`
Updated to `(a / c) + b`

Reply:
If you look changes you can see that b value is divided to c still and saved to persistent state. It is for avoid being uint256 type because b/c is specify CRS token amount which is ulong.